scss/_modal.scss

```diff
@@ -50,6 +50,28 @@
   }
 }
 
+.modal-dialog-scrollable {
+  display: flex;
+  flex-direction: column;
+  height: calc(100% - (#{$modal-dialog-margin} * 2));
+  overflow: hidden;
+
+  > .modal-content {
+    max-height: 100%;
+    overflow: hidden;
+    -ms-overflow-y: visible;
```

Comment:
What does this property do?

Reply:
For support vertically resize on IE. Resizable modal does not work well without `overflow: visible`;or `overflow: auto`. Both are OK but the behavior of the scroll bar is different.